style/js/README.md

```diff
@@ -1,3 +1,163 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [ESLint](http://eslint.org/)
+* Always use `use strict`
+
+Parens and Braces
+---
+* Leave one space between `if` and `(`.
+
+```javascript
+if (something) {
+  //some work here;
+}
+```
+
+* Prefer this form for multiline if statements.
+
+```javascript
+if (something) {
+  // do stuff
+  // even more stuff
+} else {
+  // some other stuff
+  // real stuff
+}
+```
+
+* Don't include spaces before `(` and function declaration.
+
+```javascript
+function bar() {
+  return  foo;
+}
+
+var bar = function() {
+  return foo;
+}
+```
+
+Assignments and Declarations
+---
+* Initialize arrays with `[ ]`.
+
+```javascript
+var array = [];
+```
+
+* Initialize objects with `{ }`.
+
+```javascript
+var object = {};
+```
+
+* Leave one space after `{` and before `}` when initializing objects on the same line.
+
+```javascript
+var map = { ready: 9, when: 4, "you are": 15 };
+```
+
+* Prefer single line object/array initialization.
```

Comment:
then what's the point in asking to prefer single line initialization, when it actually should be "prefer single line as long as the object is small, otherwise prefer multiline" ?